src/routes/legislative-process/+page.ts

```diff
@@ -0,0 +1,143 @@
+export interface DutySection {
+	heading: string;
+	details: string[];
+}
+
+export function load() {
+	const representatives: DutySection[] = [
+		{
+			heading: 'เสนอร่างกฎหมาย',
+			details: [
+				'สส. จำนวน 20 คน สามารถเสนอร่าง พ.ร.บ.',
+				'สส. จำนวน 1 ใน 5 เสนอแก้ไขรัฐธรรมนูญ',
+				'สส. จำนวน 1 ใน 10 สามารถเสนอร่าง พ.ร.ป.'
+			]
+		},
+		{
+			heading: 'พิจารณาร่างกฎหมาย (พ.ร.บ.)',
+			details: [
+				'ลงมติรับร่างกฎหมายเข้าสภา',
+				'พิจารณาร่างกฎหมายเป็นรายมาตราในชั้นกรรมาธิการ',
+				'ลงมติเห็นชอบหรือไม่เห็นชอบร่างกฎหมาย',
+				'ยืนยันออกร่าง พ.ร.บ. ที่ สว. ไม่อนุมัติ'
+			]
+		},
+		{
+			heading: 'อนุมัติ พ.ร.ก. หรือกฎหมายฉุกเฉินที่ออกโดย ครม. ให้เป็นกฎหมาย',
+			details: [
+				'ลงมติเห็นชอบหรือไม่เห็นชอบ พ.ร.ก.',
+				'เสนอต่อประธานสภาฯ เพื่อให้ประธานสภาฯ ส่งเรื่องไปให้ศาลรัฐธรรมนูญวินิจฉัย และให้ชะลอ พ.ร.ก.ไว้ก่อน'
+			]
+		},
```

Comment:
One heading, "อนุมัติ พ.ร.ก. หรือกฎหมายฉุกเฉินที่ออกโดย ครม. ให้เป็นกฎหมาย", is missing. Not sure if our content team just added it but please help update it.